console-subscriber/src/builder.rs

```diff
@@ -456,6 +459,56 @@ impl Builder {
     }
 }
 
+/// Specifies the address on which a [`Server`] should listen.
+///
+/// [`Server`]: crate::Server
+#[derive(Clone, Debug)]
```

Comment:
It might be desirable to mark this as 
```suggestion
#[derive(Clone, Debug)]
#[non_exhaustive]
```
on the off chance that we add more variants without breaking existing code. Although...it doesn't seem terribly likely that we will (unless we want to support Windows named pipes, or something?)

Reply:
Good idea. Done.